gcloud/credentials.py

```diff
@@ -59,6 +68,113 @@ def get_credentials():
     return client.GoogleCredentials.get_application_default()
 
 
+def _store_user_credential(credential):
+    """Stores a user credential as a well-known file.
+
+    Prompts user first if they want to store the minted token and
```

Comment:
This seems potentially confusing to me also. Would it be misleading to say "created" here?